documentation/docs/05-load.md

```diff
@@ -143,7 +143,7 @@ export async function load({ depends }) {
 - it can be used to make credentialed requests on the server, as it inherits the `cookie` and `authorization` headers for the page request
 - it can make relative requests on the server (ordinarily, `fetch` requires a URL with an origin when used in a server context)
 - internal requests (e.g. for `+server.js` routes) go direct to the handler function when running on the server, without the overhead of an HTTP call
-- during server-side rendering, the response will be captured and inlined into the rendered HTML
+- during server-side rendering, the response will be captured and inlined into the rendered HTML. Note that headers will _not_ be serialized, unless explicitly included via [`filterSerializedResponseHeaders`](/docs/hooks#handle)
```

Comment:
It was whitelisted because some tests expected it, but that seems like a bad reason to keep it. Things like `response.json()` work the same way with or without `content-type`, so the only time you'd actually _need_ it is if you're explicitly reading that header.

I did wonder about filtering the headers during SSR so that you'd get a 'btw you need to include this header' error

Reply:
(though now that I think about it i have no idea if it's possible to monkey-patch `response.headers` like that)

Reply:
the monkey has been patched